pandas_sphinx_theme/__init__.py

```diff
@@ -166,9 +171,56 @@ def get_html_theme_path():
     theme_path = os.path.abspath(os.path.dirname(__file__))
     return [theme_path]
 
+# -----------------------------------------------------------------------------
+
+# custom div directive
+# see https://github.com/dnnsoftware/Docs/tree/master/common/ext
+
+class DivNode(nodes.General, nodes.Element):
```

Comment:
Maybe can move this to a dedicated ``directives.py`` file to not have the init file too crowded?